scripts/shellcheck.sh

```diff
@@ -0,0 +1 @@
+docker run -v $(pwd):/wrk -w /wrk koalaman/shellcheck scripts/install*.sh
```

Comment:
Let's lint all the scripts:

```suggestion
docker run -v $(pwd):/wrk -w /wrk koalaman/shellcheck scripts/*.sh
```

Reply:
If we do this for all scripts it's just going to be more annoying on devs having to deal without bashisms.

You have to do BRE regex which I'm unfamiliar with. You can't use &>, etc.

I would normally consider bash as lingua franca for internal development. It was only the production install that we wanted to be more friendly.

If y'all still want it that's fine. I'm just clarifying/asking

Reply:
You can't use set -E or set -o pipefail

Reply:
You can still use shellcheck for bash/bashisms — it should pick based off of the shebang.

Reply:
Oh! Doing

Reply:
Also, in the next few months, we should start getting rid of all scripts, everything except `install_zig.sh` and `install_zig.bat`.

We should only have those to bootstrap Zig, and then move everything else into a `zig build` script. This has the advantage of working cross-platform, instead of having to create Windows versions always.